### ISSUE TITLE
HttpSnippets don't seem to be rendering the correct code

### ISSUE THREAD

**Issue by Hydragyrum** [Mermade/widdershins#283]
**Describe the bug**
A clear and concise description of what the bug is. Reports containing a full reproduction recipe (command-line and input file gist) will get a **much** faster response.

HTTP Snippets seem to render the first item on the list instead of the selected key
eg: Selecting Java with `OkHTTP` will render as `Unirest`, `httpie` renders as `curl`, etc

* [x] - I have checked that my input document is valid OpenAPI 2.0/3.0.x or AsyncAPI 1.x

**To Reproduce**
1. Command-line:
`widdershins --httpsnippet --language_tabs 'http:http' 'java:okhttp' 'shell:httpie' 'javascript:xhr' 'node:request' --summary test.yaml -o test.md`

2. My input file: (please link to a gist)
```yaml
openapi: 3.0.2
info:
  title: "Test API"
  version: "1.0.0"
  contact:
    name: Contact Name
    email: contact@example.com
    url: https://example.com/
servers:
- url: https://example.com
  description: Test Server
tags:
  - name: Generic
    description: Generic API
paths:
  /:
    get:
      operationId: getStuff
      summary: Get something
      tags: [ 'Generic' ]
      responses:
        '200':
          description: OK
```

3. Output File:
````markdown
---
title: Test API
language_tabs:
  - http: http
  - java: okhttp
  - shell: httpie
  - javascript: xhr
  - node: request
toc_footers: []
includes: []
search: true
highlight_theme: darkula
headingLevel: 2

---

<h1 id="test-api">Test API v1.0.0</h1>

> Scroll down for code samples, example requests and responses. Select a language for code samples from the tabs above or the mobile navigation menu.

Base URLs:

* <a href="https://example.com">https://example.com</a>

Email: <a href="mailto:contact@example.com">Contact Name</a> Web: <a href="https://example.com/">Contact Name</a> 

<h1 id="test-api-generic">Generic</h1>

Generic API

## Get something

<a id="opIdgetStuff"></a>

> Code samples

```java
HttpResponse<String> response = Unirest.get("https://example.com/")
  .asString();
```

```shell
curl --request GET \
  --url https://example.com/
```

```javascript
var data = null;

var xhr = new XMLHttpRequest();
xhr.withCredentials = true;

xhr.addEventListener("readystatechange", function () {
  if (this.readyState === this.DONE) {
    console.log(this.responseText);
  }
});

xhr.open("GET", "https://example.com/");

xhr.send(data);
```

```node
var http = require("https");

var options = {
  "method": "GET",
  "hostname": "example.com",
  "port": null,
  "path": "/",
  "headers": {}
};

var req = http.request(options, function (res) {
  var chunks = [];

  res.on("data", function (chunk) {
    chunks.push(chunk);
  });

  res.on("end", function () {
    var body = Buffer.concat(chunks);
    console.log(body.toString());
  });
});

req.end();
```

`GET /`

<h3 id="get-something-responses">Responses</h3>

|Status|Meaning|Description|Schema|
|---|---|---|---|
|200|[OK](https://tools.ietf.org/html/rfc7231#section-6.3.1)|OK|None|

<aside class="success">
This operation does not require authentication
</aside>
````

**Expected behavior**

I should expect to get the proper code snippets as specified at the command line

**Screenshots**
![image](https://user-images.githubusercontent.com/4928181/69948744-76705000-14f0-11ea-91b9-58abe1e72b57.png)

**Versions**:
- Node.js (note only LTS versions are supported): 
```
❯ node -v
v12.13.1
```

**Additional context**

**Comment by Hydragyrum** [comment]
Ok, closing this one because, while the documentation is a touch lacking, an environment file must be created to set the clients correctly.

Also I can't read.

**Comment by MikeRalphson** [comment]
Thanks, will look at improving the docs in this area.

**Comment by Hydragyrum** [comment]
I think it's just not very obvious that it's not possible with the command line arguments directly. The `language_tabs` option only sets the tabs and httpsnippet will just grab the defaults unless otherwise specified

**Comment by MikeRalphson** [comment]
👍

**Comment by MikeRalphson** [comment]
It's a bug, you're supposed to be able to specify `language_tabs` as an array of `language[:display[:client]]` values. Fixing now.

**Comment by MikeRalphson** [comment]
Should be fixed in version `4.0.0`, which includes breaking changes (please see the `README.md` file).
